applications/StructuralMechanicsApplication/custom_conditions/point_moment_condition_3d.cpp

```diff
@@ -0,0 +1,243 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:		 BSD License
+//					 license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Philipp Bucher
+//
+
+// System includes
+
+
+// External includes
+
+
+// Project includes
+#include "custom_conditions/point_moment_condition_3d.h"
+
+namespace Kratos
+{
+    //******************************* CONSTRUCTOR ****************************************
+    //************************************************************************************
+    
+    PointMomentCondition3D::PointMomentCondition3D( IndexType NewId, GeometryType::Pointer pGeometry )
+    : BaseLoadCondition( NewId, pGeometry )
+    {
+        //DO NOT ADD DOFS HERE!!!
+    }
+
+    //************************************************************************************
+    //************************************************************************************
+
+    PointMomentCondition3D::PointMomentCondition3D( IndexType NewId, GeometryType::Pointer pGeometry,  PropertiesType::Pointer pProperties )
+        : BaseLoadCondition( NewId, pGeometry, pProperties )
+    {
+    }
+
+    //********************************* CREATE *******************************************
+    //************************************************************************************
+
+    Condition::Pointer PointMomentCondition3D::Create(IndexType NewId,GeometryType::Pointer pGeom,PropertiesType::Pointer pProperties) const
+    {
+        return boost::make_shared<PointMomentCondition3D>(NewId, pGeom, pProperties);
```

Comment:
@philbucher Please, use the ::Pointer, we are planing to get rid of boost python  (to pybind) in a near future, and this changes will make easier to migrate

Reply:
Something like 
~~~
return Condition::Pointer( new AugmentedLagrangianMethodMortarContactCondition<TDim,TNumNodes,TFrictional, TNormalVariation> ( NewId, this->GetGeometry().Create( rThisNodes ), pProperties )); 
~~~
(I just copy paste my code, hehe)